### PR TITLE
Project Business Site: MOLECULES - Sofia Vaz Sousa

### PR DESCRIPTION
**Pull request by sofiavazs**
I managed to complete the blue, red level requirements as well as some in the black level besides the styling of the radio buttons and check boxes and BEM methodology, I also managed to do a sticky bar but not exactly as described in the black level stretch goal but it does work 😅

Initially I decided to use an image and then switched to a video because I wanted to have that challenge, used the media queries to display the right sized video in different devices.
I had a bit of trouble with the accordion but managed to make it work (in a nice way if I say so myself) after a bit of research and with help from Maks' lecture! 😊

View the site live here: https://themolecules.netlify.app/

### CODE REVIEW

File: code/style.css [hunk 1]
```diff
@@ -1,7 +1,431 @@
...
+
+body {
+  margin: 0;
+  background-color: #faf4edee;
+  font-family: "didot";
+}
+
+.herovideo video {
+  max-width:100%;
+}
```
Comment (HIPPIEKICK) [inline code/style.css:12]: Consider adding a height if you want to display only the text (if the video is not supported) as the text falls between the navbar and the form right now:
Skärmavbild 2020-09-07 kl  18 11 30

**Review by HIPPIEKICK — APPROVED** [review]
Great job Sofia! Your page looks really nice with the hero video and the sticky navbar. Nice with some validation styling, the accordion with the +/- sign that updates according to active state!

File: code/img/logo.svg [hunk 2]
```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="utf-8"?>
```
[the remark this answers is not in the document]
Comment (sofiavazs) [reply code/img/logo.svg:1]: I feel the same 😅

File: code/index.html [hunk 3]
```diff
@@ -3,28 +3,148 @@
...
+    <section class="container">
+      <h1>Book an Experimental Perfume Lab</h1>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+      <form action="http://httpbin.org/anything" method="POST">
+        <h2>Book an Experimental Perfume Lab</h2>
+        <br />
```
[the remark this answers is not in the document]
Comment (sofiavazs) [reply code/index.html:40]: Sorry! Tried to remove all the <br> following the code review last week but I missed this one! 🤦🏻‍♀️ thank you!

File: code/style.css [hunk 4]
```diff
@@ -1,7 +1,431 @@
...
+  display: flex;
+}
+
+.linkleft {
+  display: none;
+}
+
+.linkright {
+  display:none;
+}
```
[the remark this answers is not in the document]
Comment (sofiavazs) [reply code/style.css:30]: Thank you!forgot I could do that! 😅

File: code/style.css [hunk 5]
```diff
@@ -1,7 +1,431 @@
...
+
+form .radio {
+flex-direction: column;
+}
+
+form .checkbox {
+  display:flex;
+}
+
+input[type=text], select {
```
[the remark this answers is not in the document]
Comment (sofiavazs) [reply code/style.css:88]: I don't... no idea why I used it 🤦🏻‍♀️ Fixing this!

File: code/style.css [hunk 6]
```diff
@@ -1,7 +1,431 @@
...
+
+input[type=password], select {
+  width: 100%;
+  padding: 12px 20px;
+  margin: 8px 0;
+  display: inline-block;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-sizing: border-box;
+}
```
[the remark this answers is not in the document]
Comment (sofiavazs) [reply code/style.css:110]: Yes, I guess I can separate them by commas? 😅 thank you!

File: code/style.css [hunk 7]
```diff
@@ -1,7 +1,431 @@
...
+  content: '\02795';
+  font-size: 16px;
+  color: #777;
+  float: left;
+  margin-right: 10px;
+}
+
+.active:after {
+  content: "\2796";
+}
```
[the remark this answers is not in the document]
Comment (sofiavazs) [reply code/style.css:188]: Thank you ♥️

**Comment by sofiavazs** [comment]
Thank you so much Matilda! ♥️

**Pull request closed without merging**